src/using-document-creator.mdx

```diff
@@ -0,0 +1,179 @@
+---
+name: Using Document Creator
+route: /using-document-creator
+---
+
+# Using TradeTrust Document Creator
+
+You can use the [document creator](https://creator.tradetrust.io) to create your document.
```

Comment:
what does create mean in that case ? It's more than just a simple creation, documents are also issued

Reply:
ok will update it